src/features/gdpr.py

```diff
@@ -1,17 +1,18 @@
 import re
 
 from app.models import Explanation, StarCase
-from features.metadata_base import MetadataBase
-from features.website_manager import WebsiteData
-from lib.constants import STRICT_TRANSPORT_SECURITY, VALUES
+from core.metadata_base import MetadataBase
+from core.website_manager import WebsiteData
+from lib.constants import STRICT_TRANSPORT_SECURITY
 
 
+@MetadataBase.register(key="g_d_p_r")  # fixme use GDPR as key!
 class GDPR(MetadataBase):
     tag_list = ["impressum"]
     decision_threshold = 0.3
 
     @staticmethod
-    def _check_https_in_url(website_data: WebsiteData) -> list:
+    def _check_https_in_url(website_data: WebsiteData) -> list[str]:
```

Comment:
Since I see this a lot, how about using a constant for this type? Just a thought, i.e., optional

Reply:
I am not a real friend of to many type aliases. IMHO they only make sense if they have some special semantic meaning (e.g. a type alias Euro for a decimal type). Here it is just a list of strings without any extra guarantees. 

If there is special meaning it often makes sense to just use a dataclass (e.g. for a tuple[int,str,float]) instead of a type alias.

That said: For the whole values thing eventually we can define a dedicated dataclass in the future.

Reply:
Could you please create a ticket/an issue for the future to create such dedicated dataclasses for the most important return types? I keep forgetting such things, therefore I prefer to note them down :)

Reply:
Creates #80 an #81